README.md

```diff
@@ -51,29 +55,32 @@ webdav /some/path/match/* {
 }
 ```
 
-If you want to serve WebDAV and directory listing under same path (similar behaviour as in Apache and Nginx), you may use [Request Matchers](https://caddyserver.com/docs/caddyfile/matchers) to filter out GET requests and pass those to [file_server](https://caddyserver.com/docs/caddyfile/directives/file_server).
+If you want to serve WebDAV and directory listing under same path (similar
+behaviour as in Apache and Nginx), you may use [Request
+Matchers](https://caddyserver.com/docs/caddyfile/matchers) to filter out GET
+requests and pass those to
+[file_server](https://caddyserver.com/docs/caddyfile/directives/file_server).
 
 Example with authenticated WebDAV and directory listing under the same path:
 
 ```
-@notget {
-    not method GET
-}
+@get method GET
+
 route {
     basicauth {
         username hashed_password_base64
     }
-    webdav @notget
-    file_server browse
+    file_server @get browse
+    webdav
 }
 ```
 
-Or, if you want to create a public listing, but keep WebDAV behind authentication:
+Or, if you want to create a public listing, but keep WebDAV behind
+authentication:
```

Comment:
What's with the arbitrary line breaks?

Please clean up the diff and then this LGTM. :+1:

Reply:
Sorry 🙇🏽‍♂️